### ISSUE TITLE
[Enhancement] Log loss values when training

### ISSUE THREAD

**Issue by habaneraa** [Marigoldwu/A-Unified-Framework-for-Deep-Attribute-Graph-Clustering#6]
When pre-training/training the GNNs, it would be better if the loss values per step are logged. Sometimes the metric scores may not change despite the decreasing loss, and the loss values can effectively indicate the progress of self-supervised learning.

A possible solution is to add a new key-value argument (e.g. `loss=f"{loss.detach().item()}"`) when calling `logger.info(get_format_variables())` in every training script.

**Comment by Marigoldwu** [comment]
Sounds great! Thank you for your valuable suggestions and we will incorporate them in the upcoming versions. Thanks again for using this framework!

**Comment by habaneraa** [comment]
This project is helpful. I'm glad to help with building a more robust and user-friendly framework for DAGC!
